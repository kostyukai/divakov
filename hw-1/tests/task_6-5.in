0.00020626
