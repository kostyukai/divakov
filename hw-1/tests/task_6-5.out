0011111100101011000010001110111111101000101100001010111000001000
